### ISSUE TITLE
Keyboard Navigation on small viewport

### ISSUE THREAD

**Issue by zachleat** [netlify/netlify-ui-community-nav#5]
> There's a minor thing where the keyboard navigation doesn't work on smaller viewport, but I think we can go ahead and fix this once all the properties are live

https://github.com/netlify/netlify-levelup/pull/628#pullrequestreview-686684150

It does _technically_ work, but you need to open the menu with the keyboard and it’s a bit confusing.
